test/packaging/ansible/installation-newrelic-cli.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- hosts: testing_hosts
+  become: true
+  gather_facts: yes
+
+  pre_tasks:
+    - name: Initial cleanup # Only required for shared infra.
+      include_role:
+        name: cleanup
```

Comment:
do we need this?

Reply:
we still re-use infra for many test cases